docs/setup-moodle.md

```diff
@@ -1,110 +1,100 @@
+the configuration process)
 
 ### Step by step installation guide
 
-1. Log into Moodle
-2. Select the Moodle course you’d like to integrate with GitHub Classroom
-3. Turn editing on
-4. Wherever you’d like GitHub Classroom to be accessible from, click on "Add an activity or resource"
+1. Log into Moodle.
+1. Select the Moodle course to integrate with GitHub Classroom.
+1. Turn editing on.
+1. Wherever you’d like GitHub Classroom to be accessible from, click **Add an activity or resource**.
 
-    <div class="d-flex flex-justify-around">
-      <img src="/images/help/lms/moodle/step-4.png" class="border" style="width: 75%;">
-    </div>
+   <div class="d-flex flex-justify-around">
+     <img src="/images/help/lms/moodle/step-4.png" class="border" style="width: 75%;">
+   </div>
 
-5. From the dialog box which pops up, select “External tool”
+1. From the dialog box that pops up, choose **External tool** and click **Add**.
 
-    <div class="d-flex flex-justify-around">
-      <img src="/images/help/lms/moodle/step-5.png" class="border" style="width: 75%;">
-    </div>
+   <div class="d-flex flex-justify-around">
+     <img src="/images/help/lms/moodle/step-5.png" class="border" style="width: 75%;">
+   </div>
 
-6. In the dialog box which appears, enter "GitHub Classroom" for Activity Name
+1. In the _Activity name_ field, enter <kbd>GitHub Classroom</kbd>.
 
-    <div class="d-flex flex-justify-around">
-      <img src="/images/help/lms/moodle/step-6.png" class="border" style="width: 75%;">
-    </div>
+   <div class="d-flex flex-justify-around">
+     <img src="/images/help/lms/moodle/step-6.png" class="border" style="width: 75%;">
+   </div>
 
-7. For “Preconfigured Tool” click the “+” to the right of the dropdown menu.
+1. In the _Preconfigured tool_ field, click the **+** to the right of the dropdown menu. A screen appears which looks like this:
 
-8. In the following page which appears, you should see a screen which looks similar to
+   <div class="d-flex flex-justify-around">
+     <img src="/images/help/lms/moodle/step-8.png" class="border" style="width: 75%;">
+   </div>
 
-    <div class="d-flex flex-justify-around">
-      <img src="/images/help/lms/moodle/step-8.png" class="border" style="width: 75%;">
-    </div>
+   **Note**: This page may look very similar to the page you were just on. Make sure this page is titled such as _External tool configuration_ to ensure you’re on the correct one.
 
-     > Note: This page may look very similar to the page you were just on. Make sure this page says something along the lines of “External Tool Configuration” to ensure you’re on the correct one.
+1. Fill in the following information, where applicable. You may leave everything else untouched.
 
-9. Fill in the following information, where applicable. You may leave everything else untouched.
-
-    | Name | GitHub Classroom |
+    | Option | Set to |
```

Comment:
I wasn't sure if "option" was the right word here. Setting, maybe?

Reply:
Setting sounds good to me! There's no technical term for these, so do whatever you think sounds best